action.yaml

```diff
@@ -94,6 +94,10 @@ inputs:
     required: false
     default: false
     description: "Whether to checkout submodules when cloning the repository: `false` to disable (default), `true` to checkout submodules or `recursive` to recursively checkout submodules. Works only when `checkout` is set to `true`."
+  wake-on-deploy:
```

Comment:
Can we update the README as well with the new parameter, and also leave an example for the same 🙂